snuba/query/query.py

```diff
@@ -123,6 +124,12 @@ def get_offset(self) -> int:
     def set_offset(self, offset: int) -> None:
         self.__body["offset"] = offset
 
+    def get_final(self) -> bool:
+        return self.__final
+
+    def set_final(self, final) -> None:
```

Comment:
This was not here before, what is the reason for moving final into the query object instead of in the query hints/settings ?